web/src/test/java/com/epam/rft/atsy/cucumber/candidate/creation/CandidateCreationStepDefs.java

```diff
@@ -0,0 +1,207 @@
+package com.epam.rft.atsy.cucumber.candidate.creation;
+
+import static com.epam.rft.atsy.cucumber.util.DriverProvider.getDriver;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.openqa.selenium.support.ui.ExpectedConditions.textToBe;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfAllElementsLocatedBy;
+
+import com.epam.rft.atsy.cucumber.welcome.CandidateTableRow;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.ArrayList;
+import java.util.List;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+public class CandidateCreationStepDefs {
+
+  private static final String
+      CANDIDATE_CREATION_URL =
+      "http://localhost:8080/atsy/secure/candidate";
+
+  private static final String DESCRIPTION_ID = "description";
+  private static final String NAME_ID = "name";
+  private static final String EMAIL_ID = "email";
+  private static final String PHONE_ID = "phone";
+  private static final String REFERER_ID = "referer";
+  private static final String LANGUAGE_SKILL_ID = "languageSkill";
+  private static final String SAVE_BUTTON_ID = "save-button";
+
+  private static final String EMAIL_SUFFIX = "@epam.com";
+
+  private static final long TIMEOUT = 60L;
+
+  private static List<CandidateTableRow> candidateTableRows;
+
+  @Given("^other candidates data collected$")
+  public void other_candidates_data_collected() {
+    WebDriverWait webDriverWait = new WebDriverWait(getDriver(), TIMEOUT);
+
+    List<WebElement>
+        webElements =
+        webDriverWait
+            .until(visibilityOfAllElementsLocatedBy(By.cssSelector("tbody > tr[data-index]")));
+
+    candidateTableRows = new ArrayList<>();
+    for (WebElement element : webElements) {
+      List<WebElement> columns = element.findElements(By.tagName("td"));
+      String name = columns.get(0).getText();
+      String email = columns.get(1).getText();
+      String phone = columns.get(2).getText();
+      String position = columns.get(3).getText();
+
+      candidateTableRows.add(new CandidateTableRow(name, email, phone, position));
+    }
+  }
+
+  @Given("^the user is on the Candidate creation page$")
+  public void the_user_is_on_the_Candidate_creation_page() throws Throwable {
```

Comment:
Throwable is not needed, you can delete them. It was generated automatically by cucumber for the default pending exception.